homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,25 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+
+    int c;
+
+
+    while(rhs!=0){
+     c=lhs%rhs;
```

Comment:
Add space between operations like '+' '-' '!=' '==' etc...

Also what is `c`?  Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc..